src/evaluations/evaluator.py

```diff
@@ -198,7 +203,15 @@ def __init__(self, evaluation_config, sketch_estimator_config_list, run_name,
       assert isinstance(sketch_estimator_config, SketchEstimatorConfig), (
           'sketch_estimator_config_list does not contain all '
           'SketchEstimatorConfig objects')
-
+    assert isinstance(parallel_cores, int), (
+        'parallel_cores must be an int')
+    if parallel_cores < 1:
+      self.parallel_cores = cpu_count()
```

Comment:
Assuming this is actually "number of threads to use" then cpu_count() * c  where c > 1 can be nice to help account of I/O boundedness of some tasks. Typically "c" would be an input param. We don't necessarily need to add this though.

It's unclear to me after a brief look at pathos docs whether this is actually #cores or #threads though. Can you clarify?

Reply:
Well, in this case it's actually # of processes and generally the context-switching cost of processes outweighs any benefits of having more than a single one of them per core. Of course, I don't actually need to bound it and semantically it might make more sense to simply call them workers instead of cores. That might make things less arbitrarily limiting.

Reply:
Resolved in 7ea9926